packages/accounts/src/light-account/erc1271/erc1271.ts

```diff
@@ -0,0 +1,97 @@
+import type {
+  Address,
+  SmartAccountSigner,
+  ToSmartContractAccountParams,
+} from "@alchemy/aa-core";
+import { hashDomain } from "./utils/hashDomain.js";
+import { concat, hashMessage, hashTypedData } from "viem";
+import { hashType, hashStruct } from "./utils/hashTypedData.js";
+
+export interface GetErc1271SigningFunctionsParams {
```

Comment:
Why interface not type?

Reply:
I thought using `interface` when possible was best practice. [Here's an explanation](https://www.typescriptlang.org/play#example/types-vs-interfaces) from the official site that recommends interfaces and gives some reasoning why.

Reply:
oh nice, I've been wondering why I've seen uses of `interface` in a lot of alchemy's codebase and could never find a clear answer for why!